### PR TITLE
Tweak slides

### PR DESCRIPTION
**Pull request by okonet**

### CODE REVIEW

File: slides/ds101-worskhop.mdx [hunk 1]
```diff
@@ -73,22 +73,20 @@ UI developer with experience in both design and development who specializes in i
 
 ---
 
-<Grid columns={2} gap={6}>
-
 ## What this workshops is
```
Comment (varya) [inline slides/ds101-worskhop.mdx:76]: 1st level header if its separate slide?

File: slides/ds101-worskhop.mdx [hunk 2]
```diff
@@ -73,22 +73,20 @@ UI developer with experience in both design and development who specializes in i
...
 
 - Design systems overview
 - How to start a design system
 - What to include in your design system
 - Organizing design system team and processes
 
+---
+
 ## What this workshop is not
```
Comment (varya) [inline slides/ds101-worskhop.mdx:85]: Also 1st level header

**Review by varya — APPROVED** [review]
There is minor suggestion but you can merge without it.

File: slides/ds101-worskhop.mdx (hunk 1 above)
Comment (okonet) [reply slides/ds101-worskhop.mdx:76]: Doesn't really matter in this case

File: slides/ds101-worskhop.mdx (hunk 2 above)
Comment (okonet) [reply slides/ds101-worskhop.mdx:85]: I'd say no since it's a sub-section

**Pull request merged**